city_local/pdx/events/workplace-training.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: PDX October 2019 Workplace training
```

Comment:
very np, capitalize training lol? Also actually it should probably include "Organizer"